KS.Fiks.IO.Arkiv.Client.Tests/Brukerhistorier/Brukerhistorie11OppmalingsdialogTests.cs

```diff
@@ -47,6 +47,11 @@ public void TestFinnDokumentFraId()
             arkivmeldingsok.Parameter.Add(parameter);
             var payload = ArkivmeldingSerializeHelper.Serialize(arkivmeldingsok);
 
+            if (!Validator.IsValidSokXml(payload))
+            {
+                Assert.Fail("Validation errors");
+            }
+            
             Assert.Pass();
```

Comment:
`Assert.True(Validator.IsValidSokXml(payload)), "Validation errors");`

Assert.Pass er er for tidlig terminere en test og si at den er gyldig. Den har ingen verdi som siste line i en test. En test uten Assert som ikke kaster exception er en grønn test